sources/httpUtils.ts

```diff
@@ -14,11 +14,11 @@ export async function fetchUrlStream(url: string, options: RequestOptions = {})
 
   return new Promise<IncomingMessage>((resolve, reject) => {
     const request = https.get(url, {...options, agent: proxyAgent}, response => {
-      const statusCode = response.statusCode ?? 500;
-      if (!(statusCode >= 200 && statusCode < 300))
-        return reject(new Error(`Server answered with HTTP ${statusCode}`));
+      const statusCode = response.statusCode;
+      if (statusCode != null && statusCode >= 200 && statusCode < 300)
```

Comment:
@aduh95 just curious why eqeq is allowed? usually js/ts projects force eqeqeq instead of eqeq to avoid implicit conversions.

Reply:
IIRC it’s allowed only for `null`, because it’s the only value for which it’s useful instead of being confusing.